docs/specification.md

```diff
@@ -90,9 +100,11 @@ Required `Holdings` entrypoints:
 * `burnAll ()`
   * Description: burn all tokens and remove all approvals.
   * Constraints: `onlyAdmin`, `isNotPaused`.
-* `setPause (bool)`
+* `setPause (bool :value)`
   * Description: pause/unpause the contract. When paused no transactions should be possible on this contract.
   * Constraints: `onlyAdmin`.
-* `setTransferable (bool: value)`
+* `setTransferable (bool :value)`
   * Description: update `transferable` flag. When this flag is set to false, transfers should not be possible.
   * Constraints: `onlyAdmin`.
+* `getTokenMeta (view () (string, string, string))`
```

Comment:
Where did you find it?

Reply:
I've added it since I've used it in the tests

Reply:
Should I remove it from the specification?

Reply:
We may want to remove it if we get some issue with contract size or gas consumption. Anyway, I wouldn't put it into "Required `Holdings` entrypoints:", this entrypoint is not required. It's not a problem to have more entrypoints than required, but let's not add more entrypoints to this list than necessary, so that we have opportunity to drop them later if we want.

Reply:
Moved it to "Additional `Holdings` entrypoints" section